### PR TITLE
Several Package Manager fixes

### LINKED ISSUE #1565: Package Manager: "Update" button doesn't work
Clicking the Update button in the package description pane doesn't do anything. However, clicking on the update icon (up-pointing arrow) in the package list does work.

### PR DESCRIPTION
**Pull request by AlexanderSher**
- Add error notification to package manager
- Fix #1565: Package Manager: "Update" button doesn't work
- Adjust package list items height

### CODE REVIEW

File: src/R/Components/Impl/PackageManager/Implementation/ViewModel/RPackageManagerViewModel.cs [hunk 1]
```diff
@@ -68,6 +70,16 @@ internal class RPackageManagerViewModel : BindableBase, IRPackageManagerViewMode
             private set { SetProperty(ref _isLoading, value); }
         }
 
+        public string FirstError {
+            get { return _firstError; }
+            private set { SetProperty(ref _firstError, value); }
+        }
+
+        public bool HasMultipleErrors {
```
Comment (int19h) [inline src/R/Components/Impl/PackageManager/Implementation/ViewModel/RPackageManagerViewModel.cs]: Would it be easier to make this a computed property (based on `_errorMessages.Count`)?

Comment (AlexanderSher) [inline src/R/Components/Impl/PackageManager/Implementation/ViewModel/RPackageManagerViewModel.cs]: I still have to raise `PropertyChanged` event, so I'd rather change the property

**Comment by int19h** [comment]
:+1:

**Comment by huguesv** [comment]
:+1:

**Pull request merged**